addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -1277,8 +1277,16 @@ var PriorityWidget = AbstractField.extend({
     },
 });
 
-var AttachmentImage =  AbstractField.extend({
-    template: 'AttachmentImage',
+var AttachmentImage = AbstractField.extend({
+    className: 'o_attachment_image',
+    _render: function() {
```

Comment:
this should be in a private section, with docstring (@private).  Also, a space between function and (